### PR TITLE
small nits

### PR DESCRIPTION
**Pull request by patrick-ogrady**

### CODE REVIEW

File: service/service_construction.go [hunk 1]
```diff
@@ -259,11 +259,11 @@ func (s ConstructionService) ConstructionParse(
 		}
 	} else {
 		wrappedTx := new(signedTransactionWrapper)
-		if err := wrappedTx.UnmarshalJSON([]byte(req.Transaction)); err != nil {
+		if err := json.Unmarshal([]byte(req.Transaction), wrappedTx); err != nil {
 			return nil, wrapError(errInvalidInput, err)
 		}
 
-		t := new(ethtypes.Transaction)
+		var t ethtypes.Transaction
```
Comment (patrick-ogrady) [inline service/service_construction.go:266]: do this to be consistent with other uses (`*t` vs `t`)

File: service/service_construction.go [hunk 2]
```diff
@@ -128,7 +128,7 @@ func (s ConstructionService) ConstructionMetadata(
 	return &types.ConstructionMetadataResponse{
 		Metadata: metadataMap,
 		SuggestedFee: []*types.Amount{
-			mapper.FeeAmount(suggestedFee, input.Currency), //TODO: LOOK AT!
+			mapper.AvaxAmount(big.NewInt(suggestedFee)),
```
Comment (patrick-ogrady) [inline service/service_construction.go:131]: Fee is never not AVAX

**Review by dhrubabasu — APPROVED** [review]
LGTM, the pointer fixes should be done in the main PR @learyce

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [ master ]
   pull_request:
-    branches: [ master ]
```
Comment (dhrubabasu) [inline .github/workflows/ci.yml:7]: <3

**Review by learyce — APPROVED** [review]

**Pull request merged**